index.js

```diff
@@ -42,6 +42,67 @@ function set(obj, path, value) {
   obj[components[components.length - 1]] = value;
 }
 
+function find(list, predicate) {
+  // 1. Let O be ? ToObject(list value).
+  if (list == null) {
+    throw new TypeError('"list" is null or not defined');
```

Comment:
Not sure if `TypeError` is the correct one to throw here but I guess it doesn't matter... :)